README.md

```diff
@@ -1,5 +1,10 @@
 # Apache Beam starter for Java
 
+ℹ️ This repository is dual licensed on both [Apache ASL2](LICENSE-APACHE) and [MIT](LICENSE-MIT) licenses.
```

Comment:
I believe the ASF requires the Apache license?
"The ASF will not dual-license our software because such licenses make it impossible to determine the conditions under which we have agreed to collaborate on a collective product, and are thus contrary to the Apache spirit of open, collaborative development among individuals, industry, and nonprofit organizations."

https://www.apache.org/licenses/GPL-compatibility.html

Reply:
We asked this to legal and the fast summary is that we're okay with dual licensing to make it easier for users to choose whichever license they prefer. This is just a template for users, not a product or project by itself. The license here is more for contributors to this repo and we just have to make that explicit on our contributing guides.

Here's the legal thread for more information:
https://issues.apache.org/jira/browse/LEGAL-601

Reply:
Did you pull the verbiage of CONTRIBUTING.md and LICENSE from existing projects? It would be great to use something canonical.

Reply:
Do you have any reference existing project with a verbiage you prefer?

Reply:
I'd suggest putting this boilerplate in the README: 

```
License

This software is distributed under the terms of both the MIT license and the
Apache License (Version 2.0).

See LICENSE for details.
```

And using this for the headers:

```
Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
<LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
option. This file may not be copied, modified, or distributed
except according to those terms.
```
For Google OSS projects, I don't think you're required to mention the MIT or Apache license in the CONTRIBUTING file.

Reply:
Thanks! I've added the boilerplate to the README and used the headers for every other source file.